liberty-maven-plugin/src/it/tests/loose-config-it/pom.xml

```diff
@@ -138,6 +138,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-failsafe-plugin</artifactId>
+                <version>2.5</version>
```

Comment:
I just use the same version as in install-apps-project-it.  I will change it to 2.19.1